Assignment01/step-by-step.md

```diff
@@ -35,7 +35,11 @@ Now you can test whether you can call the VehicleRegistrationService. You can do
 
 1. Click on `Send request` in the file to send a request to the API:
 
-   ![REST client](img/rest-client.png)
+   ```http
```

Comment:
I'm not sure this is helpful. The screenshot was very clear for those who could read it, as it pointed out where exactly that button is located. The textual representation of the screenshot lost that info. I would say a screenshot makes more sense here than a textual representation of it.

Reply:
(also applies to the Java guide, by the way)

Reply:
Did you not notice the awesome sauce ASCII art arrow that indicates where the `Send request` button is????

No, I agree. I will put the screenshot back.